generators/fastlane-setup/templates/env

```diff
@@ -0,0 +1,16 @@
+# Build number is composed of :
```

Comment:
Why grouping this in a single file? 
What if you have to manage different apps with different versions? (example: Storyssimo + Instant Parisien)

Reply:
Then you split it up